pkg/adapter/metrics.go

```diff
@@ -27,11 +30,12 @@ const (
 
 // Label kinds supported by mixer.
 const (
-	String LabelKind = iota
+	String LabelType = iota
```

Comment:
They are the same, we can use the proto types directly I suppose. However, I think we want a separation between our API layer and our internal representation of that API for an aspect, which is why this enum exists.

If we want to remove this enum lets do it in a follow-on PR, since it requires updating both of the metric aspects too.